src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Organize CockroachDB Cloud Clusters Using Folders
+summary: Learn how to use folders to organize CockroachDB Cloud clusters and manage access to them.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/limited-access.md %}
+
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters using folders. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}](managing-access.html).
+
+## How folders work
+
+By default, clusters are created at the level of the organization. Folders provide additional ways to organize clusters and manage access to them. A folder can contain both clusters and descendent folders.
```

Comment:
nit: remove the word descendant because it seems like the focus of this sentence is what the children of a folder can be

Reply:
Done